docroot/themes/humsci/humsci_basic/humsci_basic.info.yml

```diff
@@ -4,8 +4,8 @@ description: A base for visually distinctive HumSci themes.
 package: 'Humanities & Sciences'
 core: 8.x
 core_version_requirement: ^8
-libraries:
-  - humsci_basic/base
+libraries: 
+  # Libraries are loaded via sub-themes
```

Comment:
I believe this will cause an error. either comment out the `libraries` or set it to an empty array `libraries: { }`